<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustCNVEditor</class>
 <widget class="QWidget" name="CustCNVEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>变量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCNVPath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCfg">
       <property name="text">
        <string>变量配置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCNVIn"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCNVOut">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
